docs/local_resource.md

```diff
@@ -43,6 +44,31 @@ but not in response to any particular file changes.
 As with `docker_build` and `custom_build`, you can specify files/directories to be
 ignored [with the `ignore` argument](http://tilt.dev/2019/06/07/better-monorepo-container-builds-with-context-filters.html).
 
+## Resource dependencies
+
+The `resource_deps` argument allows you to specify the order that resource deps initialize.
+
+A common pattern is to use `local_resource` to run database initialization after
+your database comes up in Kubernetes.
+
+See the [Resource Dependencies guide](resource_dependencies.html) for more.
+
+## Parallelism
+
+Local resources can read and write to the local filesystem freely. 
+
+Tilt will run local resource in serial, to ensure that a file write from one resource doesn't
+interfere with another local resouce.
```

Comment:
```suggestion
By default, local resources will not update at the same time as any other resources, to ensure that any files it writes don't cause problems. (e.g., infinitely re-triggering itself, or triggering another resource when its input is only half-ready).
```
1. worth clarifying that this only applies to update cmds, not run cmds
2. they're exclusive with all other resources, not just local resources
3. if we get more explicit about the problems this prevents, readers are more likely to be able to make a good decision about whether they want to use `allow_parallel`
  a. Maybe we should write more than what I put in parens in the suggestion.
  b. Maybe we should also say something like "if your local resource writes no files, this is always safe to enable".

Reply:
hmmm...i don't think your new text is accurate. how would reducing parallelism stop infinite loops?

Reply:
> hmmm...i don't think your new text is accurate. how would reducing parallelism stop infinite loops?

Yeah, I think you're right. I'm not sure what case I was thinking of.